.storybook/docs/HtmlElementArgs.tsx

```diff
@@ -37,7 +36,7 @@ export const HtmlElementArgs = ({ attributes, of }) => {
 
   return (
     <>
-      <h3 id="additional-props">Additional {attributes ? 'attributes' : 'props'}</h3>
```

Comment:
This block didn't really apply to our web components because spreading any prop doesn't actually filter through the host web component element. As a result, we've created these attr lists with pre-defined attributes one might want to pass along to an internal element. 

So I removed the "attribute" language from this component and created a new one to work for web component docs.